content/docs/overview.md

```diff
@@ -3807,6 +3844,36 @@ n := halve(4).? or_else 0
 fmt.println(n)                   // 2
 ```
 
+#### Testing
+
+Odin includes [a "core:testing" package](https://pkg.odin-lang.org/core/testing/) for unit testing purposes.
+
+A minimalist example test:
+
+```odin
+// tests/test_code.odin
+
+package test_code
+
+import "core:testing"
+
+@test
+test_that_hello_world_equals :: proc(t: ^testing.T) {
+	expected := "Hello, World!"
+
+	// More likely this is a call to a function you want to test
+	result := "Goodbye, Mars!"
+
+	testing.expect_value(t, result, expected)
+}
+```
+
+You can run this test using `odin test tests/`. Any tests in the given directory that are annotated with `@test` will be run. You can also run a single test file using `odin test tests/test_code.odin -file`.
+
+You may also use the [`_test` file suffix](#file-suffixes) to only include a file in the build when testing. e.g: `foo_test.odin`. Just like other file suffixes, this means that the file will only be analyzed when you are running `odin test`.
+
+If you'd like to refer to good example tests, see [Odin/tests](https://github.com/odin-lang/Odin/tree/master/tests) and [ols/tests](https://github.com/DanielGavin/ols/tree/master/tests).
```

Comment:
As good as it is, Odin can't take credit for the language server 😛 
Might be worth a rephrase a long the lines of:
> If you'd like to refer to good example tests, see [Odin/tests](https://github.com/odin-lang/Odin/tree/master/tests).
> The [unofficial language server](https://github.com/DanielGavin/ols/tree/master/tests) also has tests which could be could be worth a look.

Might be worth getting Bill's blessing on mentioning OLS in the overview though.

Reply:
That is fine.

Reply:
As long as it is made extremely clear that it is "unofficial", then sure.

The main reason for this is because people will be posting issues in the wrong place otherwise.